cmd/xfn/spark/spark.go

```diff
@@ -126,12 +129,20 @@ func (c *Command) Run() error { //nolint:gocyclo // TODO(negz): Refactor some of
 		return errors.Wrap(err, errParseRef)
 	}
 
+	opts := []oci.ImageClientOption{FromImagePullConfig(req.GetImagePullConfig())}
+	if c.CABundlePath != "" {
+		rootCA, err := xpkg.ParseCertificatesFromPath(c.CABundlePath)
```

Comment:
I don't love `internal/oci` taking a dependency on `internal/xpkg` just for this function:

* Eventually I think we'll want `internal/xpkg` to depend on `internal/oci` - xpkgs are built on OCI images.
* Nothing about `ParseCertificatesFromPath` is specific to `xpkg` functionality.

Is there somewhere else `ParseCertificatesFromPath` could live? Given the function is pretty simple I'd be fine just duplicating it in this package if there's not an obvious place for it to live right now.

Reply:
perhaps `internal/transport`?

Reply:
Ah, you're right. I mistakenly thought this file was in `internal/oci` for some reason. I'd be okay with:

* Creating `internal/transport` as you suggest. Maybe premature for just one function though.
* Maybe just move the function into `internal/oci`? Rationale being OCI is more generic than xpkg
* Just proceed as you have it now and we figure it out later. 🙂